### DISCUSSION TITLE
Skin editor, object list should be re-positioned depending on location

### DISCUSSION THREAD

**Discussion by nyaruku** [ppy/osu#14962] [Ideas] [answered]
![osu_2021-10-05_19-32-03](https://user-images.githubusercontent.com/38096703/136076628-b6cfc89d-fc2e-44c8-b948-aca59263ed68.png)

Cant scroll throught list if it's for example on the bottom.

**Comment by gagahpangeran** [comment, marked as answer]
https://github.com/ppy/osu/issues/12749
